### PR TITLE
Add net2cog service definition

### PR DESCRIPTION
**Pull request by frankinspace**
## Jira Issue ID

N/A

## Description

Added service definition for net2cog

## Local Test Steps

See https://github.com/podaac/net2cog/pull/28

Also added regression test https://github.com/nasa/harmony-regression-tests/pull/94

## PR Acceptance Checklist
* [ ] Acceptance criteria met
* [x] Tests added/updated (if needed) and passing
* [ ] Documentation updated (if needed)

### CODE REVIEW

**Comment by frankinspace** [comment]
Please deploy https://github.com/podaac/net2cog/pkgs/container/net2cog/266279787?tag=0.4.0rc2 to UAT

File: config/services.yml [hunk 1]
```diff
@@ -851,6 +851,31 @@ https://cmr.uat.earthdata.nasa.gov:
         - image/tiff
       reprojection: true
 
+  - name: net2cog
```
Comment (indiejames) [inline config/services.yml:854]: should this be 'podaac/net2cog'?

File: services/harmony/test/versions.ts [hunk 2]
```diff
@@ -34,6 +34,7 @@ describe('Versions endpoint', function () {
           'sds/variable-subsetter',
           'sds/maskfill',
           'sds/trajectory-subsetter',
+          'net2cog',
```
Comment (indiejames) [inline services/harmony/test/versions.ts:37]: Same question as before: should this be 'podaac/net2cog'?

File: config/services.yml (hunk 1 above)
Comment (frankinspace) [reply config/services.yml:854]: Not sure, is the `podaac/` part significant?

File: config/services.yml [hunk 3]
```diff
@@ -851,6 +851,31 @@ https://cmr.uat.earthdata.nasa.gov:
...
+          <<: *default-turbo-env
+          STAGING_PATH: public/harmony/net2cog
+    umm_s: S1268612173-POCLOUD
+    capabilities:
+      subsetting:
+        variable: true
+      output_formats:
+        - image/tiff
+        # Don't know how specific/accuate this should be. This is (as far as I can tell) the 'official' mime-type that specifies COG. Got this value from https://github.com/opengeospatial/CloudOptimizedGeoTIFF/issues/1 and from pystac media type constant https://pystac.readthedocs.io/en/stable/api/media_type.html
+        - image/tiff; application=geotiff; profile=cloud-optimized
```
Comment (chris-durbin) [inline config/services.yml:872]: Harmony ignores everything after the semicolon so this string will match against any user request that includes `image/tiff` like `image/tiff; profile=foo`. So you can delete this entry - maybe keep in a comment about the official COG mimetype in here if it's useful.

File: config/services.yml (hunk 1 above)
Comment (chris-durbin) [reply config/services.yml:854]: The only places this gets exposed I believe is in our documentation at harmony.earthdata.nasa.gov/docs 
image and our /versions endpoint. It's up to you if you want to include podaac there or not - it is not needed.

**Review by chris-durbin — APPROVED** [review]
Tested successfully in sandbox along with harmony-ci-cd changes.

File: config/services.yml (hunk 1 above)
Comment (frankinspace) [reply config/services.yml:854]: I'm fine leaving it off

**Review by ygliuvt — APPROVED** [review]

**Pull request merged**